blockchain/message_pool/src/msgpool/utils.rs

```diff
@@ -47,6 +47,7 @@ pub(crate) async fn recover_sig(
     let val = bls_sig_cache
         .get(&msg.cid()?)
         .ok_or_else(|| Error::Other("Could not recover sig".to_owned()))?;
-    let smsg = SignedMessage::new_from_parts(msg, val.clone()).map_err(Error::Other)?;
+    let smsg = SignedMessage::new_from_parts(msg, val.clone())
+        .map_err(|err| Error::Other(err.to_string()))?;
```

Comment:
We already have this:
```rust
impl From<anyhow::Error> for Error {
    fn from(e: anyhow::Error) -> Self {
        Error::Other(e.to_string())
    }
}
```

```suggestion
    let smsg = SignedMessage::new_from_parts(msg, val.clone())?;
```

Reply:
fixed